core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -80,6 +80,8 @@ class ReligionManager {
 
     fun endTurn(faithFromNewTurn: Int) {
         storedFaith += faithFromNewTurn
+        if (civInfo.isPlayerCivilization())
+            storedFaith += faithFromNewTurn*200
```

Comment:
:thinking:

Reply:
Damn and I thought I got rid of all the debug code. :(